app/lib/realm.js

```diff
@@ -370,6 +384,7 @@ class DB {
 		serversDB: new Realm({
 			path: 'default.realm',
 			schema: [
+				userSchema,
```

Comment:
Can we move this schema to `activeDB`?

Reply:
Hi @diegolmello, We can't move this schema to ```activeDB``` because on ```handleSelectServer``` we need to access user before to ```setActiveDB```.